bin/measureHscRC2Metrics.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+
+pipetask --long-log run -j $NUMPROC -b SMALL_HSC/butler.yaml -d "detector in (58, 50, 42, 47, 49, 41) AND visit in (29336, 11690, 11698, 29350, 11696, 11704, 11710, 11694, 1220, 1204, 23694, 1206, 23706, 23704, 1214, 23718, 19694, 19680, 30490, 1242, 19684, 30482, 19696, 1248, 1178, 17948, 17950, 17904, 1184, 17906, 17926, 17900, 11738, 358, 11724, 346, 22632, 11740, 22662, 322)" -p 'pipelines/DRP.yaml#singleFrame' -i HSC/RC2/defaults --register-dataset-types -o jenkins/singleFrame
```

Comment:
If you are processing all the data, why are you specifying a `-d` at all?  You can just leave it out and all the data will be processed.  This is true throughout.

Reply:
I wasn't certain that the list of visits/detectors from the "notes.txt" file in `gen3_rc2_subset` included everything. Now that you've confirmed that is the case, I've removed the `-d` from the script.